.circleci/config.yml

```diff
@@ -46,19 +76,61 @@ job-references:
       - run:
           name: "Run Tests"
           command: |
-            if [[ "$RUN_EXTRA" == "1" ]]; then
-              npm run lint
-            fi
-            rm -rf $WP_TESTS_DIR $WP_CORE_DIR
-            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 $WP_VERSION
+            rm -rf << pipeline.parameters.wp_core_dir >> << pipeline.parameters.wp_test_dir >>
+            bash bin/install-wp-tests.sh << pipeline.parameters.db_name >> << pipeline.parameters.db_user >> "<< pipeline.parameters.db_pass>>"  << pipeline.parameters.db_host >> "$WP_VERSION"
             vendor/bin/phpunit --version
             vendor/bin/phpunit
 
+  php_core_job: &php_core_job
+    steps:
+      - checkout
+      - run: *setup_environment
+      - run: *install_dependencies
+      - run: *prepare_repo
+      - run:
+          name: "Run Tests"
+          command: |
+            rm -rf << pipeline.parameters.wp_core_develop_dir >>
+            bash bin/install-wp-develop.sh << pipeline.parameters.db_name >> << pipeline.parameters.db_user >> "<< pipeline.parameters.db_pass>>"  << pipeline.parameters.db_host >> "$WP_VERSION"
+            cd << pipeline.parameters.wp_core_develop_dir >>
+            npm ci
+            npm run grunt build
+            export WP_TESTS_DIR=<< pipeline.parameters.wp_core_develop_dir >>/tests/phpunit
+            echo "test EN"
+            echo $WP_TESTS_DIR
+            ~/project/vendor/bin/phpunit --log-junit << pipeline.parameters.phpunit_report_file >> || true
+            ~/project/vendor/bin/phpunit --bootstrap ~/project/tests/bootstrap.php --log-junit << pipeline.parameters.phpunit_report_file_including_mu_plugins >> || true
+            diff <(cat << pipeline.parameters.phpunit_report_file >> | grep -o '<testsuite name="".*time=') <(cat << pipeline.parameters.phpunit_report_file_including_mu_plugins >> | grep -o '<testsuite name="".*time=')
+
+
+  lint_job: &lint_job
```

Comment:
I have moved out lining to a separate more lightweight = faster job, which is independent to the core tests work